plugin-server/src/main/pluginsServer.ts

```diff
@@ -428,6 +433,18 @@ export async function startPluginsServer(
             }
         }
 
+        if (capabilities.personOverrides) {
+            const postgres = hub?.postgres ?? new PostgresRouter(serverConfig)
+            const kafkaProducer = hub?.kafkaProducer ?? (await createKafkaProducerWrapper(serverConfig))
+
+            personOverridesPeriodicTask = new DeferredPersonOverrideWorker(postgres, kafkaProducer).runTask(5000)
+            personOverridesPeriodicTask.promise.catch(async () => {
```

Comment:
Exposing the `promise` here seems kind of leaky but not so much that I want to forward `catch` through as `PeriodicTask.catch` right now.